package.json

```diff
@@ -38,7 +38,7 @@
     "storybook": "lerna run storybook --scope @serge/components --stream",
     "build-storybook": "lerna run build-storybook --scope @serge/components --stream",
     "build": "lerna run build --scope @serge/components --scope @serge/config --scope @serge/helpers --scope @serge/store --scope @serge/client --stream",
-    "pkg-build": "lerna run start --scope @serge/executable --stream",
```

Comment:
Hmm, it did look like we were previously telling lerna to just build the executable.  Wouldn't that give better performance?

Reply:
not sure but issue was causing due to --scope flag.